wellcomecollection/wellcomecollection.org
# Added logic to set exhibition format type in hardcopy subheadings

## What does this change?

This adds logic to allow the hardcopy in `Exhibiton.tsx` to render the correct exhibition format type in its subheadings. Permanent exhibitions are treated the same as exhibitions (who have no format type) and will render 'E/exhibition' where it is called. 

This change comes from #10985, where there is a need to display "display" instead of "exhibition" where appropriate in the page text. 

## How to test

Many of the other types (with exception to Installation, which has its own page component - `Installation.tsx`) do not (yet) refer to the hardcopy subheadings as in a standard temporary exhibition page. A way to test is to add the code to the hardcoded 'Visit us' with the logic: 
```TypeScript
title={`Visit us - THIS is a ${exhibitionFormat}!`}
```
which should render this (depending on the page type you test this on): 
![Screenshot 2024-07-08 at 17 27 48](https://github.com/wellcomecollection/wellcomecollection.org/assets/81175151/fde9e900-b40f-4937-854a-2f8355565929)

## How can we measure success?

When options are added in Prismic - their hardcopy will reflect in the page type correctly. 

## Have we considered potential risks?

I am presuming no Prismic conflicts as these are hardcopy changes. Do correct / save the site otherwise!

## Note
`.toLowerCase` is required to ensure the subheading remains grammatically correct in that instance.
